### ISSUE TITLE
Accounts.addEmail and Accounts.createUser - better docs / add feature / bug

### ISSUE THREAD

**Issue by alan-pl** [meteor/meteor#10746]
Accounts.addEmail and Accounts.createUser -- these API methods allow the email argument to be any non-empty string. This is not intuitive. I suggest one or more of these fixes: 
- document this behavior (simplest, easiest)
- a server-side config setting could be available, like "rfcEmailsOnly: boolean", or "emailValidator: function", etc.
- this could even be regarded as a bug, on the grounds that "addEmail" implies that the argument must be a valid email address. Maybe the methods should run a stricter validation, but this seems like a breaking change.

**Comment by filipenevola** [comment]
Hi @alan-pl, can you open a PR against docs updating with more info in these methods https://github.com/meteor/docs? 

Feel free to also open a PR against the accounts package providing new options.

**Comment by alan-pl** [comment]
Can do. Now I'm wondering if this will all be solved by the introduction of TypeScript... Will the docs become typescript-y?

**Comment by filipenevola** [comment]
Yes, Typescript can help but I believe the docs will be useful in the future anyway, I would not wait for Typescript to have better docs.

I think you should go forward with a PR 😉
